django/utils/timezone.py

```diff
@@ -295,13 +295,15 @@ def template_localtime(value, use_tz=None):
 
 # Utilities
 
-def localtime(value, timezone=None):
+def localtime(value=None, timezone=None):
```

Comment:
Yes, that's what I was suggesting.

Distinct PR / commits aren't a big deal. In fact, if I was writing the code, I'd create two commits, one to improve `localtime` first and then one to add `localdate` but that doesn't matter much anyway.